package.json

```diff
@@ -30,7 +30,7 @@
     "@dojo/has": "beta1",
     "@dojo/i18n": "beta1",
     "@dojo/shim": "beta1",
-    "maquette": "2.4.3"
+    "maquette": "2.5.0"
```

Comment:
It just makes changes like this breaking as consumers will need to upgrade their Maquette version. I'd be happier if we actually specified Maquette as a real dependency (as it is), so consumers would just get the version of Maquette that is compatible with the version of `widget-core`